### ISSUE TITLE
Duplicate References in DblPendulum

### ISSUE THREAD

**Issue by Ant13731** [JacquesCarette/Drasil#2773]
I think this may be the result of how the recipes are made, but the logs I'm working on (in #2759) showed some duplicate references using the angular displacement instance model in dblpendulum under the requirements section:
![image](https://user-images.githubusercontent.com/69334555/128733455-d56bb3b3-df5c-48ec-825d-922d9903f751.png)
The first statement should probably be grouped so that the two variables are listed as
```
θ and θ_p (from IM:calOfAngularDisplacement)
```
and the second statement should not reference the same variable from the same instance model twice.

**Comment by Ant13731** [comment]
Keeping this issue open to make sure we check for other occurrences of things like this.

**Comment by JacquesCarette** [comment]
Yes, it would be nice to at least identify if there are other such occurrences and make issues.
